### ISSUE TITLE
Can you help me?

### ISSUE THREAD

**Issue by boy-new-2019** [ifzhang/FairMOT#124]
I tried to use fairmot for multiple targets, but unfortunately I found that num_class has been modified after training, the effect is not good. Then I found that you said to modify the number of channels in the Heatmap head. Where can I modify the number of channels in the Heatmap head?

**Comment by ifzhang** [comment]
https://github.com/ifzhang/FairMOT/blob/ddc9200289cb9a52ec640f0d6999b7ff5c601391/src/lib/opts.py#L217
You can change opt.num_classes. The channel is set here.

**Comment by boy-new-2019** [comment]
> https://github.com/ifzhang/FairMOT/blob/ddc9200289cb9a52ec640f0d6999b7ff5c601391/src/lib/opts.py#L217
> 
> 
> 您可以更改opt.num_classes。在这里设置频道。

yes ,i change the num_class in jde.py to 13，and then I think the 【'wh': 2 if not opt.cat_spec_wh else 2 * opt.num_classes,】also need to change , so I set it to 26,however it meets some error.

Latter,  I keep the set about【'wh': 2 if not opt.cat_spec_wh else 2 * opt.num_classes,】.and change the num_class to 13，but i find it seems has no work.
